src/main/java/org/opensearch/sdk/rest/ExtensionRestPathRegistry.java

```diff
@@ -171,9 +190,18 @@ public List<String> getRegisteredDeprecatedPaths() {
      *
      * @param method  the method.
      * @param path  the path.
+     * @param name  the name corresponding to this route.
+     * @param actionNames the set of actions registered with this route
      * @return A string appending the method and path.
      */
-    public static String restPathToString(Method method, String path) {
-        return method.name() + " " + path;
+    public static String restPathToString(Method method, String path, String name, Set<String> actionNames) {
+        StringBuilder sb = new StringBuilder(method.name() + " " + path + " ");
+        if (name != null && !name.isEmpty()) {
+            sb.append(name).append(" ");
+        }
+        if (actionNames != null) {
```

Comment:
Is it enforced anywhere that `actionNames` cannot contain spaces? If space is being used as a delimiter than an `actionName` must not contain a space

Reply:
I had this same question and [yes, it is](https://github.com/DarshitChanpura/OpenSearch/blob/bea26f97c66070581070764fe4790d2f0579c8e7/server/src/main/java/org/opensearch/rest/NamedRoute.java#L27).